modules/docs/markdown/06-guides/testing.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: Testing
+title: Testing Smithy4s Applications
+---
+
+In this guide, we will give you some guiding principles and other things to consider when testing Smithy4s applications.
+
+## What Smithy4s Does
+
+For starters, let's briefly cover what it is that Smithy4s does. At a high level, Smithy4s takes your Smithy model and generates Scala code based on that model. The model is essentially just a Scala representation of the Smithy model. The generated code _does not_ contain any logic for any specific library such as http4s or jsoniter. Instead, it contains abstractions that Smithy4s itself defines that interpreters can be based on. This means that the generated code is not coupled to any library (other than Smithy4s itself). The interpreters that we have created in Smithy4s will then take the generated code and use it to power, for example, and HTTP service that uses http4s and jsoniter.
```

Comment:
**issue:** `and HTTP service` -> `an HTTP service`

we all need more Grammarly in our lives

Reply:
Hah for real!